client.go

```diff
@@ -105,8 +101,8 @@ type Dialer interface {
 // NewClient returns a Client instance ready for communication with the given
 // server endpoint. It will use the latest remote API version available in the
 // server.
-func NewClient(endpoint string) (*Client, error) {
-	client, err := NewVersionedClient(endpoint, "")
+func NewClient(nodes []string) (*Client, error) {
```

Comment:
maybe: ```func NewClient(endpoint node...) (*Client, error) {``` ?

Reply:
Would that let you specify "host1,host2,host3" in a string (no client changes, or would it need "host1", "host2", "host3" (client changes)?

Can/should we allow both?

Reply:
I guess we could allow both, good argument for supporting env variables out of the box. It could then try splitting the strings inside while still allowing client to specify endpoints explicitly.

Reply:
I will update this PR to make the method variadic